lighthouse-core/gather/gatherers/html-without-javascript.js

```diff
@@ -39,17 +41,16 @@ class HTMLWithoutJavaScript extends Gatherer {
     // Reset the JS disable.
     options.disableJavaScript = false;
 
-    const driver = options.driver;
-
-    this.artifact = {};
-    return driver.evaluateAsync(`(${getBodyText.toString()}())`)
+    return options.driver.evaluateAsync(`(${getBodyText.toString()}())`)
       .then(result => {
-        this.artifact = result;
+        this.artifact = {
+          value: result
```

Comment:
Seems like in all other gatherers, we assign the result directly to `this.artifact`. Why in this case do we add a value property?